### PR TITLE
Create releases via hub

### PR DESCRIPTION
**Pull request by kevgo**
These releases contain only the files we want to publish, and contain them sorted alphabetically.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -63,6 +63,26 @@ lint-go:  # lints the Go files
...
 	tools$/prettier$/node_modules$/.bin$/prettier -l .
 
+release:   # creates a new release
+	# cross-compile the binaries
+	goreleaser --rm-dist
+
+	# make Windows installer
+	$(MAKE) msi
+
+  # create GitHub release with files in alphabetical order
```
Comment (charlierudolph) [inline Makefile:73]: Indent here is off

File: Makefile [hunk 2]
```diff
@@ -63,6 +63,26 @@ lint-go:  # lints the Go files
...
+	hub release create --draft --browse --message v7.4.0 \
```
Comment (charlierudolph) [inline Makefile:74]: Why is it a draft release? And what is browse for?

Comment (kevgo) [reply Makefile:74]: I'm creating draft releases that I manually review and publish until this script is battle-tested enough that I trust to run it fully automatically. Once a release is out there, it's impossible to remove it anymore.

**Comment by kevgo** [comment]
Example release: https://github.com/git-town/git-town/releases/tag/untagged-adde9a69d1377b70f42d

File: Makefile (hunk 1 above)
Comment (kevgo) [reply Makefile]: Thanks for catching! Fixed.

File: Makefile (hunk 2 above)
Comment (kevgo) [reply Makefile:74]: `--browse` opens the new release in a browser window for inspection and publishing.

File: Makefile [hunk 3]
```diff
@@ -63,6 +63,26 @@ lint-go:  # lints the Go files
...
+	# cross-compile the binaries
+	goreleaser --rm-dist
+
+	# make Windows installer
+	make --no-print-directory msi
+
+	# create GitHub release with files in alphabetical order
+	hub release create --draft --browse --message v7.4.0 \
+		-a dist/git-town_7.4.0_linux_intel_64.deb \
+		-a dist/git-town_7.4.0_linux_intel_64.rpm \
```
Comment (kevgo) [inline Makefile:76]: I have decided to omit 32 bit builds. 32 bit is seriously outdated at this point. 32 bit computers can only use 2 GB RAM. All hardware has been 64 bit for over 10 years. Even Linux distros are giving up on maintaining 32 bit versions. Even cell phones have been 64 bit for several generations now. Given that 64 bit OS can run 32 bit apps, there is no reason to use a 32 bit OS anymore. Hence no reason for 32 bit Git Town binaries.

File: Makefile [hunk 4]
```diff
@@ -63,6 +63,26 @@ lint-go:  # lints the Go files
...
+		-a dist/git-town_7.4.0_linux_intel_64.tar.gz \
+		-a dist/git-town_7.4.0_linux_arm_64.deb \
```
Comment (kevgo) [inline Makefile:78]: Omitting support for cell phone ARM processors (ARM v5-v7).

File: Makefile [hunk 5]
```diff
@@ -63,6 +63,26 @@ lint-go:  # lints the Go files
...
+		-a dist/git-town_7.4.0_linux_arm_64.rpm \
+		-a dist/git-town_7.4.0_linux_arm_64.tar.gz \
+		-a dist/git-town_7.4.0_macOS_intel_64.tar.gz \
```
Comment (kevgo) [inline Makefile:81]: Omitting support for BSD. BSD is mostly used on servers these days. I want to see actual need from the community for a BSD release before adding these binaries. I think its better if the list of releases is concise and easy to skim.

**Pull request merged**